src/mm/guestmem.rs

```diff
@@ -222,3 +222,31 @@ impl<T: Sized + Copy> GuestPtr<T> {
         unsafe { GuestPtr::from_ptr(self.ptr.offset(count)) }
     }
 }
+
+mod tests {
+
+    #[test]
+    fn test_read_u8_valid_address() {
```

Comment:
This ends up using inline assembly, so Miri cannot handle it.

```suggestion
    #[test]
    #[cfg_attr(miri, ignore)]
    fn test_read_u8_valid_address() {
```

Reply:
Not sure if this is really needed anymore if we do `#[cfg_attr(test_in_svsm, ignore = "<REASON>")]` but including just in case